ci-operator/step-registry/openshift/e2e/openstack/sriov/openshift-e2e-openstack-sriov-workflow.yaml

```diff
@@ -11,6 +11,7 @@ workflow:
       - ref: optional-operators-subscribe
       - ref: openstack-provision-sriov-networknodepolicy
```

Comment:
Is it normal that it's missing the `openstack-provision-sriov-operator` step?

Reply:
yes because the operator is not deployed from OLM or source, but from the CI step registry in `optional-operators-subscribe`. So we don't need `openstack-provision-sriov-operator`.